static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/DetailsContent/DetailsContent.tsx

```diff
@@ -143,6 +146,31 @@ const DetailsContent = ({ selectedId }: Props) => {
             title: isBlender ? "Users" : "Machines",
             value: subscription.number_of_machines,
           },
+          ...(isBlender
+            ? // Don't show the column for Blender subscriptions.
+              []
+            : [
+                {
+                  title: "Active machines",
+                  value: (
+                    <React.Fragment>
+                      {subscription.number_of_active_machines}
+                      <Tooltip
+                        tooltipClassName="p-subscriptions-tooltip"
+                        message="The number of machines that contacted Ubuntu Advantage in the last 24 hour period"
```

Comment:
(Beta)

Reply:
@wgx made the change, you can check on the demo and if it's good we can merge.